python/ql/test/experimental/dataflow/coverage/test.py

```diff
@@ -1,20 +1,23 @@
-# This should cover all the syntactical constructs that we hope to support
+# This should cover all the syntactical constructs that we hope to support.
+# Headings refer to https://docs.python.org/3/reference/expressions.html,
+# and are selected whenever they incur dataflow.
 # Intended sources should be the variable `SOURCE` and intended sinks should be
 # arguments to the function `SINK` (see python/ql/test/experimental/dataflow/testConfig.qll).
 #
 # Functions whose name ends with "_with_local_flow" will also be tested for local flow.
+#
+# All functions starting with "test_" should run and print a source (sources are defined in testConfig.qll).
 
-# These are included so that we can easily evaluate the test code
+# These are defined so that we can evaluate the test code.
 SOURCE = "source"
```

Comment:
How about a global `NONSOURCE = "not a source"` and use that to indicate a value that's not supposed to be considered as a source, instead of throwing in `3` or `42j` etc?

Reply:
We could do that. I like how SOURCE and SINK stand out in the code now, though. It allows the brain to pick out those two important pieces. Non-sources seem lees important to me; there are also many other non-flow things happening...

Reply:
Actually, let us try it, it may not look to bad :-) (also, it is basically only `3`, `42j` is the complex source )

Reply:
Done